tests/e2e/volumepropverification_test.go

```diff
@@ -123,7 +123,7 @@ func TestVolumeProperties(t *testing.T) {
 				}
 				log.Println("Finished verifying volume properties like size, disk-format and attached-to-vm fields"+
 					" at vm and esx for volume - ", volName)
-				verification.ExecCmd(vms[vmIndx], "docker stop "+containerName+" ; docker rm "+containerName)
+				ssh.InvokeCommand(vms[vmIndx], "docker stop "+containerName+" ; docker rm "+containerName)
```

Comment:
Prefer to make this a function to say DeleteRunningContainer() which will stop and then remove the container. And it should be in the dockercli package. Else this model will allow tests to create scripts inside of a call to invokeCommand.